docker/Dockerfile

```diff
@@ -99,13 +100,6 @@ WORKDIR /root/code/garage
 COPY .pre-commit-config.yaml /root/code/garage
 RUN ["/bin/bash", "-c", "source activate garage && git init && pre-commit"]
 
-# Pull CodeClimate reporter
-RUN curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > /root/code/garage/cc-test-reporter && \
-  chmod +x /root/code/garage/cc-test-reporter
-
 # Add code stub last
 COPY . /root/code/garage
 RUN ["/bin/bash", "-c", "source activate garage && pip install -e /root/code/garage && source deactivate garage"]
```

Comment:
the `pip install -e` step should be done in the Dockerfile, not the entrypoint

Reply:
But this is the Dockerfile :confused: